### PR TITLE
Add hover states for inspector icons

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by anthonydillon** [comment]
### QA
- Add a service
- Check the toggle icons in the inspector are not visible
- Hover over the server name and see the icons fade in to `opacity: .6`
- Then hover over the toggle icons and see that they transition to `opacity: 1`

File: jujugui/static/gui/src/app/components/added-services-list/_added-services-list.scss [hunk 1]
```diff
@@ -49,6 +51,8 @@
     }
 
     &__item-name {
+        position: relative;
+        top: 2px;
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/added-services-list/_added-services-list.scss]: These small tweaks to the icons are just to vertically align them

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3849/
Test PASSed.

**Comment by makyo** [comment]
:+1: QA :ok:  This is minor, so :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**